sportngin/verification
# Support rails 6.0, 6.1 and Ruby 2.7.8, 3.0.5

What
----------------------
### Major version update! 3.0.0 Removes tested support for Ruby < 2.6.1

> Uprading to support rails 6.0, 6.1, and ruby 2.7.8, 3.0.5

Why
----------------------
> Progress, advancement, manifest destiny
- Part of Ngin Rails 6.0 upgrade

Deploy Plan
-----------
> Nothing special.

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
> https://sportngin.atlassian.net/browse/CPO-5214

QA Plan
-------
(Will need to be followed up with a PR for ngin to pull new version and check it on staging for errors)
